notes/2025/2025-07-01-contributor-pool.md

```diff
@@ -0,0 +1,146 @@
+   * **Time to merge:** 1 day, 1 hour
+   * **Effort Estimate:** 2
+
+2. [fix: handle URL-encoded link fragments in no-missing-link-fragments rule](https://github.com/eslint/markdown/pull/437)
+   The purpose of this pull request is to fix a bug in the no-missing-link-fragments rule where URL-encoded link fragments were not correctly matched to their corresponding headings. The changes include decoding URL-encoded link fragments before comparison to avoid false positives. 
+   * **Time to merge:** 3 days, 1 hour
+   * **Effort Estimate:** 3
+
+3. [feat: add no-bare-urls rule](https://github.com/eslint/markdown/pull/418)
+   This PR implements the `no-bare-urls` rule to enforce proper URL formatting in Markdown documents. It ensures that URLs are either wrapped in angle brackets (autolinks) or used with text (links), instead of being used as bare URLs. 
+   * **Time to merge:** 8 days, 5 hours
+   * **Effort Estimate:** 4
+
+4. [feat: add no-invalid-named-grid-areas rule](https://github.com/eslint/css/pull/169)
+   This pull request adds a new rule `no-invalid-named-grid-areas` to enforce valid named grid area definitions in CSS Grid templates. It helps prevent common mistakes when using grid-template-areas by checking that named grid areas are properly defined according to the CSS Grid specification. 
+   * **Time to merge:** 18 days, 6 hours
+   * **Effort Estimate:** 4
+
+5. [feat: add no-reversed-media-syntax rule](https://github.com/eslint/markdown/pull/398)
+   This PR adds a new rule `no-reversed-media-syntax` to enforce correct link and image syntax in Markdown. The rule checks for and prevents reversed media syntax. 
+   * **Time to merge:** 19 days, 11 hours
+   * **Effort Estimate:** 3
+
+6. [feat: add checkSiblingsOnly option to no-duplicate-headings rule](https://github.com/eslint/markdown/pull/393)
+   The purpose of this pull request is to add a new `siblingsOnly` option to the no-duplicate-headings rule to make it more flexible, allowing it to only check for duplicate headings among siblings. 
+   * **Time to merge:** 7 days, 1 hour
+   * **Effort Estimate:** 3
+
+7. [fix!: remove rollup and extraneous types](https://github.com/eslint/markdown/pull/383)
+   This pull request simplifies the build process and type management by removing unnecessary Rollup bundling for this ESM package. It discusses potential breaking changes related to type declarations. 
+   * **Time to merge:** 31 days, 2 hours
+   * **Effort Estimate:** 5
+
+8. [feat: add no-empty-definitions rule](https://github.com/eslint/markdown/pull/392)
+   This PR adds a new rule `no-empty-definitions` to ensure that both regular definitions and footnote definitions in Markdown files are meaningful and not empty. 
+   * **Time to merge:** 8 days, 5 hours
+   * **Effort Estimate:** 3
```

Comment:
here the PR link is incorrect - https://github.com/eslint/markdown/pull/364

Also the author is Pixel998 and not xbinaryx